packages/block-library/src/image/styles.native.scss

```diff
@@ -31,22 +31,53 @@
 	align-items: center;
 }
 
+.imageUploadingIconContainer {
+	width: 40px;
+	height: 40px;
+	justify-content: center;
+	align-items: center;
+}
+
 .iconRetry {
 	fill: #fff;
 	width: 100%;
 	height: 100%;
 }
 
-.icon {
+.iconPlaceholder {
 	fill: $gray-dark;
 	width: 100%;
 	height: 100%;
 }
 
-.iconDark {
+.iconPlaceholderDark {
 	fill: $white;
 }
 
+.iconUpload {
+	fill: $gray-lighten-20;
+	width: 100%;
+	height: 100%;
+}
+
+.iconUploadDark {
+	fill: $gray-70;
+	width: 100%;
+	height: 100%;
```

Comment:
In the dark version of the style we don't need to re-define other than the color properties. All properties that are the same will be copied from the light version.

(Same for all other Dark style definitions)

Reply:
Ah I see, thanks! Updated f32aab37dcdaaeeaa1ae16dd1e8b8a87bab8b001